pycbc/transforms.py

```diff
@@ -1468,6 +1468,389 @@ def from_config(cls, cp, section, outputs):
+            out[self.latitude_ssb_param], out[self.polarization_ssb_param] = \
+            coordinates.geo_to_ssb(
+                maps[self.tc_geo_param], maps[self.longitude_geo_param],
+                maps[self.latitude_geo_param], maps[self.polarization_geo_param]
+                )
+        return self.format_output(maps, out)
+
+    def inverse_transform(self, maps):
+        """This function transforms arrival time, sky localization,
+        and polarization angle in the SSB frame to the corresponding
+        values in the geocentric frame.
+
+        Parameters
+        ----------
+        maps : a mapping object
+
+        Returns
+        -------
+        out : dict
+            A dict with key as parameter name and value as numpy.array or float
+            of transformed values.
+        """
+        out = {}
+        out[self.tc_geo_param], out[self.longitude_geo_param], \
+            out[self.latitude_geo_param], out[self.polarization_geo_param] = \
+            coordinates.ssb_to_geo(
+                maps[self.tc_ssb_param], maps[self.longitude_ssb_param],
+                maps[self.latitude_ssb_param], maps[self.polarization_ssb_param]
+                )
+        return self.format_output(maps, out)
+
+    @classmethod
+    def from_config(cls, cp, section, outputs):
+        tag = outputs
+        skip_opts = []
+        additional_opts = {}
+
+        # get custom variable names
+        variables = {
+            'tc-geo': cls.default_params_name['_default_tc_geo'],
+            'longitude-geo': cls.default_params_name['_default_longitude_geo'],
+            'latitude-geo': cls.default_params_name['_default_latitude_geo'],
+            'polarization-geo': cls.default_params_name['_default_polarization_geo'],
+            'tc-ssb': cls.default_params_name['_default_tc_ssb'],
+            'longitude-ssb': cls.default_params_name['_default_longitude_ssb'],
+            'latitude-ssb': cls.default_params_name['_default_latitude_ssb'],
+            'polarization-ssb': cls.default_params_name['_default_polarization_ssb']
+        }
+        for param_name in variables.keys():
+            name_underline = param_name.replace('-', '_')
+            if cp.has_option("-".join([section, outputs]), param_name):
+                skip_opts.append(param_name)
+                additional_opts.update(
+                    {name_underline+'_param': cp.get_opt_tag(
+                     section, param_name, tag)})
+            else:
+                additional_opts.update(
+                    {name_underline+'_param': variables[param_name]})
+
+        return super(GEOToSSB, cls).from_config(
+            cp, section, outputs, skip_opts=skip_opts,
+            additional_opts=additional_opts
+        )
+
+
+class LISAToSSB(BaseTransform):
+    """Converts arrival time, sky localization, and polarization angle in the
+    LISA frame to the corresponding values in the SSB frame."""
+
+    name = "lisa_to_ssb"
+    default_params_name = {
```

Comment:
@ahnitz I have a question here, you can see the default names of `lisa` and `ssb` are the same; will this cause some potential issues if the user doesn't set custom names for them? Last time in PE slack channel, you both mentioned not hard code det-related names.

Reply:
Yes, it might. Perhaps we should require that the user set these, rather than make an assumption? If we do find a convention that allows an obvious default we can always put one in.